Tests/Realm.Tests/Sync/AsymmetricObjectTests.cs

```diff
@@ -341,7 +341,8 @@ public void EmbeddedObject_WhenParentAccessed_ReturnsParent()
             SyncTestHelpers.RunBaasTestAsync(async () =>
             {
                 var flxConfig = await GetFLXIntegrationConfigAsync();
-                flxConfig.Schema = new[] {
+                flxConfig.Schema = new[]
+                {
```

Comment:
I also figured I should fix this since the warning annoyed me

Reply:
This was also fixed by @elle-j in [#3064](https://github.com/realm/realm-dotnet/pull/3064/files)

Reply:
thanks for the heads up, I can remove this from my change to avoid unnecessary merge conflict